installer/inventory

```diff
@@ -72,3 +72,6 @@ pg_port=5432
 #http_proxy=http://proxy:3128
 #https_proxy=http://proxy:3128
 #no_proxy=mycorp.org
+
+# Container configuration
```

Comment:
Possibly rename this to `Container networking configuration` and then in the future we can re-use this section if we had other related configurations.